packages/checkout/components/text-input/validated-text-input.tsx

```diff
@@ -254,7 +256,7 @@ const ValidatedTextInput = forwardRef<
 				onBlur={ () => validateInput( false ) }
 				ariaDescribedBy={ describedBy }
 				value={ value }
-				title=""
+				title="" // This prevents the same error being shown on hover.
```

Comment:
This was only mentioned in the original PR https://github.com/woocommerce/woocommerce-blocks/pull/8143#discussion_r1065711272, it's better near the code now.